modules/helm/crds.mk

```diff
@@ -37,29 +37,29 @@ ifeq ($(HOST_OS),darwin)
 	sed_inplace := sed -i ''
 endif
 
+crds_dir ?= config/crd/bases
+
 .PHONY: generate-crds
 ## Generate CRD manifests.
 ## @category [shared] Generate/ Verify
 generate-crds: | $(NEEDS_CONTROLLER-GEN) $(NEEDS_YQ)
-	$(eval crds_gen_temp := $(bin_dir)/scratch/crds)
 	$(eval directories := $(shell ls -d */ | grep -v -e 'make' $(shell git check-ignore -- * | sed 's/^/-e /')))
 
-	rm -rf $(crds_gen_temp)
-	mkdir -p $(crds_gen_temp)
+	mkdir -p $(crds_dir)
```

Comment:
I agree with comments on the previous PR that this no longer belongs in the helm makefile-module. But we could do this refactoring as a separate change? WDYT? My knowledge for makefile-modules is limited, so if I am going to do this refactoring, someone needs to show me how to declare makefile-module dependencies. Any examples already present?

Reply:
I think we do really need more docs around makefile-modules

I'd be _fairly_ comfortable leaving this in the helm module with a TODO comment, but I worry that might actually make it more difficult to migrate later - it's unlikely, but if someone wanted to use an older version of the Helm module that supported these CRDs, they wouldn't be able to do so with the new crds module we'd create later.

I think for that reason I'm softly in favour of moving this to a different place in this PR, but it's not a blocker for me (and I don't currently have bandwidth to help with it so it's very easy for me to say that 😂 )

Reply:
I'm in favour of incremental improvements. I'd say keep it here, add a TODO / issue for the wider change.

Reply:
Seems like we agree to make this initial change smaller and improve afterwards.